onnxruntime/core/optimizer/gelu_fusion.h

```diff
@@ -17,10 +17,16 @@ x * 0.5 * (1.0 + erf(x / sqrt(2.0))), where x is the input.
 
 */
 class GeluFusion : public GraphTransformer {
+ private:
+  int optimize_level = 1;
+
  public:
   GeluFusion(const InlinedHashSet<std::string_view>& compatible_execution_providers = {}) noexcept
       : GraphTransformer("GeluFusion", compatible_execution_providers) {}
 
+  GeluFusion(int level,
+             const InlinedHashSet<std::string_view>& compatible_execution_providers = {}) noexcept;
+
```

Comment:
Can we have a single ctor with the level defaulting to 1 and use TransformerLevel instead of `int` for the level?

Reply:
Yeah, I will fix it.